packages/react/src/components/CardGroup/CardGroup.js

```diff
@@ -35,11 +35,15 @@ const CardGroup = ({ cards, cta }) => {
       const { current: containerNode } = containerRef;
       if (containerNode) {
         sameHeight(
-          containerNode.getElementsByClassName(`${prefix}--card__heading`),
+          containerNode.getElementsByClassName(
+            `:not(.bx--card__video) ${prefix}--card__heading`
```

Comment:
@ariellalgilmore We should probably use the `${prefix}` token here.

Reply:
oh yeah... i knew I was missing something